src/core/Internal.ml

```diff
@@ -1735,11 +1735,11 @@ module Make (Plugin : PLUGIN) = struct
             (let lits, proof = mk_expl () in
              let l = List.rev_map (fun f -> Atom.neg @@ mk_atom st f) lits in
              (* note: we can check that invariant here in the [lazy] block,
-             as conflict analysis will run in an environment where
-             the literals should be true anyway, since it's an extension of the
-             current trail
-             (otherwise the propagated lit would have been backtracked and
-             discarded already.) *)
+                as conflict analysis will run in an environment where
+                the literals should be true anyway, since it's an extension of the
+                current trail
+                (otherwise the propagated lit would have been backtracked and
+                discarded already.) *)
```

Comment:
Here ocamlformat seems to have done the right thing, I'm starting to think there's some weird rules for comments, or maybe that running ocamlformat once does not reach a fixpoint on comment indentation ?